amd_openvx/openvx/CMakeLists.txt

```diff
@@ -183,3 +183,25 @@ else()
     target_link_libraries(openvx dl m)
     target_link_libraries(vxu openvx)
 endif()
+
+if(BUILD_FILE_REORG_BACKWARD_COMPATIBILITY)
+  include(ROCMHeaderWrapper)
+  # Copy Header files to build directory
+  configure_file("${PROJECT_SOURCE_DIR}/include/vx_ext_amd.h" "${PROJECT_BINARY_DIR}/include/mivisionx/vx_ext_amd.h" COPYONLY)
```

Comment:
change ${PROJECT_BINARY_DIR}/include/mivisionx/vx_ext_amd.h" to ${PROJECT_BINARY_DIR}/include/vx_ext_amd.h"